termine/announce.go

```diff
@@ -42,10 +42,11 @@ func init() {
 func announceStammtisch(t *data.Termin) error {
 	maildraft := `Liebe Treffler,
 
-am kommenden Donnerstag ist wieder Stammtisch. Diesmal sind wir bei {{.Location}}.
+am kommenden Donnerstag ist wieder Stammtisch. Diesmal sind wir bei
+{{.Location}}.
```

Comment:
[solution of the problem for historical documentation](https://github.com/nnev/website/commit/d73c95a128cf55c9d25166f0c20e7c001e8bec93)